packages/app/src/app/pages/Dashboard/Content/routes/RecentSandboxes/index.js

```diff
@@ -1,66 +0,0 @@
-import React from 'react';
-import { observer, inject } from 'app/componentConnectors';
-import Helmet from 'react-helmet';
-
-import { Query } from 'react-apollo';
-
-import getMostUsedTemplate from '../../../utils/get-most-used-template';
-
-import { Content as Sandboxes } from '../../Sandboxes';
-
-import CreateNewSandbox from '../../CreateNewSandbox';
-import { RECENT_SANDBOXES_CONTENT_QUERY } from '../../../queries';
-
-const RecentSandboxes = ({ store }) => (
-  <>
-    <Helmet>
-      <title>Recent Sandboxes - CodeSandbox</title>
-    </Helmet>
-    <Query
-      variables={{
-        orderField: store.dashboard.orderBy.field,
-        orderDirection: store.dashboard.orderBy.order.toUpperCase(),
-      }}
-      query={RECENT_SANDBOXES_CONTENT_QUERY}
-    >
-      {({ loading, error, data }) => {
-        if (error) {
```

Comment:
You are missing this check in the tsx file

Other than that looks great!

Reply:
thanks Sara! fixed